### ISSUE TITLE
How could i run demo if input-image as cv2_image format?

### ISSUE THREAD

**Issue by hungtooc** [clovaai/deep-text-recognition-benchmark#245]
in demo.py, they import input images as file:
`demo_data = RawDataset(root=opt.image_folder, opt=opt)  # use RawDataset
    demo_loader = torch.utils.data.DataLoader(
        demo_data, batch_size=opt.batch_size,
        shuffle=False,
        num_workers=int(opt.workers),
        collate_fn=AlignCollate_demo, pin_memory=True)` 
how can i input cv2_image?
Thank for a great project.
